tools/transformer/src/atlasTransformations.js

```diff
@@ -40,5 +40,21 @@ module.exports = function runTransformations(openapi) {
     ignoredModelNames
   );
 
+  // Temp workaround for
+  // https://jira.mongodb.org/browse/CLOUDP-166120
+  openapi.components.responses.noBody = {
+    content: {
+      "application/vnd.atlas.2023-01-01+json": {
+        example: "",
+      },
+    },
+    description: "This endpoint does not return a response body",
+  };
```

Comment:
this was resolved upstream but missed deployment, can you please backport @matt-condon

Reply:
No need for backport as we are using dev version of the sdk for that work with other fixes. 
But since both dev and prod use the same codebase it is better to keep workarounds for little bit longer. 
I have jira to remove them so we good.

Reply:
do you have the ticket?